lib/src/blockdev.rs

```diff
@@ -50,6 +50,24 @@ impl Device {
         self.children.as_ref().map_or(false, |v| !v.is_empty())
     }
 
+    // Return true if device is mounted anywhere
+    pub(crate) fn is_mounted_in_pid_mounts(&self) -> Result<bool> {
+        let output = Command::new("findmnt")
+            .arg("-N")
+            .arg("1")
+            .arg("--output=SOURCE")
+            .output()
+            .expect("Failed to execute findmnt");
+
+        let mounts = String::from_utf8(output.stdout).unwrap();
```

Comment:
While `findmnt` giving non-utf8 output is a "shouldn't happen" case, it's equally easy for us to use `?` here to propagate it as an error instead of aborting, so let's do that.

Reply:
How about adding this code to `mount.rs` instead? We could aim to share code with `run_findmnt`...at least in terms of parsing JSON instead of text.

Reply:
Just pushed up a refactor into `mount.rs`, it now shares some code with the other findmnt functions in there. Is that more in line with what you were thinking?